app/controllers/proxy_types_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class ProxyTypesController < ApplicationController
+  def index
+    @result = proxy_types.where!
```

Comment:
it'd be cool to have a method on the repo interface called `all!` that just calls `where!` without params.  just an alias really, but it'd be more readable in situations like this.  definitely not in scope here, just a note for the future

Reply:
Let's do it! This makes total sense.